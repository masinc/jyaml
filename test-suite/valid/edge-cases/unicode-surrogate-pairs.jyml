# Unicode surrogate pairs for 4-byte characters (JYAML 0.4 spec)
"emoji": "🚀🎉🦀"
"surrogate_pairs": "\uD83D\uDE80\uD83C\uDF89\uD83E\uDD80"
"mixed": "Hello 🌍 World"
"copyright": "© 2023"