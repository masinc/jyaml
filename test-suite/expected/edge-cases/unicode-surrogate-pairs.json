{"emoji": "🚀🎉🦀", "surrogate_pairs": "🚀🎉🦀", "mixed": "Hello 🌍 World", "copyright": "© 2023"}